assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterServletModule.java

```diff
@@ -32,24 +32,26 @@ protected void configureServlets() {
       install(new org.eclipse.che.core.tracing.web.TracingWebModule());
     }
 
-    final Map<String, String> corsFilterParams = new HashMap<>();
-    corsFilterParams.put("cors.allowed.origins", "*");
-    corsFilterParams.put(
-        "cors.allowed.methods", "GET," + "POST," + "HEAD," + "OPTIONS," + "PUT," + "DELETE");
-    corsFilterParams.put(
-        "cors.allowed.headers",
-        "Content-Type,"
-            + "X-Requested-With,"
-            + "accept,"
-            + "Origin,"
-            + "Access-Control-Request-Method,"
-            + "Access-Control-Request-Headers");
-    corsFilterParams.put("cors.support.credentials", "true");
-    // preflight cache is available for 10 minutes
-    corsFilterParams.put("cors.preflight.maxage", "10");
-    bind(CorsFilter.class).in(Singleton.class);
+    if (Boolean.valueOf(System.getenv("CHE_WSMASTER_CORS_FILTER_ENABLED"))) {
```

Comment:
Env name is not compliant with other env names we have. Consider `CHE_CORS__FILTER_ENABLED` or `CHE_CORS__FILTER__ENABLED`

Reply:
can I specify anyhow that this related to WS Master's (Che server) CORS filter, and not WS Agent?

Reply:
che.properties contains examples of our properties naming. We don't use any specific part for wsmaster but we do use it for wsagent. I would follow this pattern.